api/api/settings.py

```diff
@@ -195,6 +195,14 @@
 }
 
 # user management
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+        'OPTIONS': {'min_length': 8}
+    },
+    {'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator'},
+    {'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator'},
```

Comment:
Doesn't this introduce more password rules than just the min length?

Reply:
yes - I figured that doesn't hurt, although is non-critical functionality, so I didn't write a test. I'm fine dropping it. What do you think?

Reply:
It should be either dropped or documented

Reply:
dropped